CHANGELOG.md

```diff
@@ -14,6 +14,16 @@ The main work (all changes without a GitHub username in brackets in the below li
 -   @matter/nodejs
     - Feature: New export @matter/nodejs/config allows for fine-grained configuration of Node.js bootstrap logic
 
+-   @matter/create
+    - Feature: Add command line option "--verbose" to enable informational NPM messages during initialization
+    - Feature: Add template "contributor" to bootstrap dev environment for working on matter.js itself
```

Comment:
Should be also add this to the main README explicitely or into CONTRIBUTING.md or such?

PS: And maybe also add the withFeatures/withBehaviors to changelog :-)

Reply:
K I added the changelog entry and will wait on updating README/CONTRIBUTING pending testing once released.